src/components/Image/Image.md

```diff
@@ -1,10 +1,8 @@
-The `Image` component renders an img element within a wrapper div. This div is displayed as inline-block so it behaves like a standard inline image. Height and width actually get applied to the wrapper div, and the image will get scaled/positioned within the wrapper depending on which props you provide.
```

Comment:
looks like you're removing all the top-level component descriptions, are they going somewhere else?

Reply:
Yeah, I was looking at the Styleguidist examples, and they should be a comment on the class itself in the implementation file.